Utilities/constants.py

```diff
@@ -0,0 +1,34 @@
+"""
+All of the constant values used across all of the scripts are located here.
+"""
+
+# Safety
+DEFAULT_VELOCITY = 0.50
+VELOCITY_THRESHOLD = 2  # never let the drone go faster than 1 m/s for safety (is this a good number?)
+MINIMUM_ALLOWED_ALTITUDE = 0.5
+MAXIMUM_ALLOWED_ALTITUDE = 4.0
+RANGEFINDER_MIN = 0.29
+RANGEFINDER_EPSILON = 0.03
+
+# Connection-oriented
+CONNECTION_STRING_SIMULATOR = "tcp:127.0.0.1:5762"
+CONNECTION_STRING_REAL = "/dev/serial/by-id/usb-3D_Robotics_PX4_FMU_v2.x_0-if00"
+
+# DroneKit Vehicle Modes
+GUIDED_MODE = "GUIDED"
+LAND_MODE = "LAND"
+
+# Directions
+UP = (0, 0, 1)
```

Comment:
This is the perfect case for a enumation class like the following:

```
from enum import Enum
class Directions(Enum):
    UP = (0, 0, 1)
    # ...
```

Note that you will have to use `Directions.U.value` to get the original tuple value